packages/auth/__tests__/providers/cognito/confirmSignInHappyCases.test.ts

```diff
@@ -29,9 +32,9 @@ const authConfig = {
 const mockedGetCurrentUser = getCurrentUser as jest.Mock;
 
 describe('confirmSignIn API happy path cases', () => {
-	let handleChallengeNameSpy;
-	const username = authAPITestParams.user1.username;
-	const password = authAPITestParams.user1.password;
+	let handleChallengeNameSpy: jest.SpyInstance;
+	const { username } = authAPITestParams.user1;
+	const { password } = authAPITestParams.user1;
```

Comment:
Nit: Here and below and in other files
```suggestion
	const { password, username } = authAPITestParams.user1;
```

Reply:
this might have been auto fix run.

Reply:
Hum I'm if there is a lint rule for this and auto fixable, will look

Reply:
Didn't find a rule for this for auto fix. Manually fixed.